mreg_cli/network.py

```diff
@@ -276,17 +283,22 @@ def find(args: Namespace):
             print_network_info(nwork)
             print()  # Blank line between networks
 
-    s = "s" if n_networks > 1 else ""
     if not args.silent:
+        s = "s" if n_networks > 1 else ""
         print(f"Found {n_networks} network{s} matching the search criteria.")
 
 
 network.add_command(
     prog="find",
-    description="Search for networks based on a range of search parameters.",
-    short_desc="Search for networks.",
+    description="Search for networks based on a range of search parameters",
+    short_desc="Search for networks",
     callback=find,
     flags=[
+        Flag(
+            "-ip",
+            description="Exact host address",
```

Comment:
Might not be a host. Just an ip.

Reply:
[af2db9f](https://github.com/unioslo/mreg-cli/pull/159/commits/af2db9ff18cba715ed3f9518c10db8dbc662711f)